source/before-integrating/register-and-manage-your-service.html.md.erb

```diff
@@ -0,0 +1,44 @@
+---
+title: Register and manage your service
+weight: 5.13
+last_reviewed_on: 2024-09-25
+review_in: 6 months
+---
+
+# Register and manage your service
+
+You get a unique client ID when you register your service. You'll need this client ID to integrate each of your services with GOV.UK One Login. 
+
+You should configure multiple client IDs if you have multiple environments, for example staging, user acceptance testing, integration, production. There's further [guidance on creating a configuration for each service you’re integrating][integrate.client-id].
+
+Registering should take 5 minutes to complete. To register your service to use GOV.UK One Login, you'll need: 
+
+* a government email address
+* a mobile phone
+
+If you do not have a government email address or mobile phone, you should find a civil servant in your team who can register the service on your behalf.
+
+Whoever registers the service will have the entry tied to their email address. It is currently not possible to reassign ownership if someone leaves or to add multiple email addresses to a particular client. If you need access after someone has left, you can create an additional client using a different email address and transfer the configuration settings to the new account. 
+
+1. [Create an admin tool account][external.self-service-admin].
+1. Enter your email address and confirm your email security code.
```

Comment:
```suggestion
1. Go to the [Get started with GOV.UK One Login][[external.get-started-gov-uk-one-login] page and select *Create admin tool account*.
1. Then, follow the on-screen instructions to enter your email address and confirm your email security code.
```

Reply:
looks like there are double left brackets in `[[external.get-started-gov-uk-one-login]` should be `[external.get-started-gov-uk-one-login]`